### PR TITLE
Bugfix/1.8.0 fix deb python version

### PR DESCRIPTION
**Pull request by ShayNehmad**
# What is this? 
Update out .deb package python version to 3.7. 

This removes our (OOTB) support for Ubuntu 16 and lower.

## Checklist
* [X] Have you added an explanation of what your changes do and why you'd like to include them?
* [x] Have you successfully tested your changes locally?
* [x] Is the TravisCI build passing?

### CODE REVIEW

File: monkey/monkey_island/deb-package/DEBIAN_MONGO/postinst [hunk 1]
```diff
@@ -1,20 +1,42 @@
...
+# This is using the apt package `python3.7-venv` which is listed in the `control` file as a dependency.
+# See https://packages.debian.org/stable/python/python3.7-venv
+echo "Using $(command -v $PYTHON_VERSION) as the base for virtualenv creation"
+$PYTHON_VERSION -m venv ${PYTHON_FOLDER}
+# shellcheck disable=SC1090
+source ${PYTHON_FOLDER}/bin/activate
+
+echo "Installing Python dependencies using $(command -v pip)..."
+# First, make sure that pip is updated
+${PYTHON_FOLDER}/bin/python -m pip install --upgrade pip
```
Comment (danielguardicore) [inline monkey/monkey_island/deb-package/DEBIAN_MONGO/postinst:29]: If you ran source above, why are you fully qualifying your python command?

File: monkey/monkey_island/deb-package/DEBIAN_MONGO/postinst [hunk 2]
```diff
@@ -25,11 +47,17 @@ if [ -d "/etc/systemd/network" ]; then
...
 service monkey-mongo start
 
-echo Monkey Island installation ended
+echo ""
+echo "Monkey Island installation ended."
+echo "The server should be accessible soon via https://<server_ip>:5000/"
+echo "To check the Island's status, run 'sudo service monkey-island status'"
+echo ""
 
 exit 0
```
Comment (danielguardicore) [inline monkey/monkey_island/deb-package/DEBIAN_MONGO/postinst:63]: Newline

File: monkey/monkey_island/linux/create_certificate.sh [hunk 3]
```diff
@@ -2,8 +2,21 @@
...
 server_root=${1:-"./cc"}
 
+echo "Creating server cetificate. Server root: $server_root"
+# We override the RANDFILE determined by default openssl.cnf
+# This is a known issue with the current version of openssl on Ubuntu 18.04 - once they release
+# a new version, we can delete this command. See
+# https://github.com/openssl/openssl/commit/0f58220973a02248ca5c69db59e615378467b9c8#diff-8ce6aaad88b10ed2b3b4592fd5c8e03a
+# for more details.
+dd bs=1024 count=2 </dev/urandom >~/.rnd
+chmod 666 ~/.rnd
```
Comment (danielguardicore) [inline monkey/monkey_island/linux/create_certificate.sh:12]: And if you don't have permissions to ~ or if the file exists?

File: monkey/monkey_island/requirements.txt [hunk 4]
```diff
@@ -1,5 +1,4 @@
 pytest
-bson
```
Comment (danielguardicore) [inline monkey/monkey_island/requirements.txt:2]: YAY

Comment (itaymmguardicore) [inline monkey/monkey_island/requirements.txt:2]: I would also like to express my happiness of removing this annoying package

File: monkey/monkey_island/linux/create_certificate.sh (hunk 3 above)
Comment (ShayNehmad) [reply monkey/monkey_island/linux/create_certificate.sh]: 9687b22b

The user which runs ./create_cert will have access to their own ~ folder in enough of the cases that I'm willing to leave this in. No other workaround that doesn't involve carrying openssl with us or messing with worse things in the system that I could fine.

File: monkey/monkey_island/deb-package/DEBIAN_MONGO/postinst (hunk 1 above)
Comment (ShayNehmad) [reply monkey/monkey_island/deb-package/DEBIAN_MONGO/postinst]: 592fd887

File: monkey/monkey_island/deb-package/DEBIAN_MONGO/postinst (hunk 2 above)
Comment (ShayNehmad) [reply monkey/monkey_island/deb-package/DEBIAN_MONGO/postinst]: 592fd887

**Pull request merged**